assembl/graphql/idea.py

```diff
@@ -425,6 +428,12 @@ def resolve_posts(self, args, context, info):
                 query = query.options(
                     models.LangString.joinedload_option(Post.body))
 
+        from_node = args.get('from_node')
+        if from_node and not args.get('after') and not args.get('before'):
+            post_id = int(Node.from_global_id(from_node)[-1])
+            node_idx = len(list(takewhile(lambda post: post[0] != post_id, query.with_entities(Post.id))))
+            args['after'] = offset_to_cursor(node_idx - 1)
```

Comment:
How were you able to find this method to use?

Reply:
I dig into the code ;)